File: python codes/linearSearch.py
def linearSearch(arr, l, e):
    for i in range(0,l):
        if arr[i]==e:
            return i
    return -1
a=[2,54,62,64,22]
n=len(a)
x=int(input("Enter the element to be searched: "))
result=linearSearch(a,n,x)
if result==-1:
    print("Element does not exist in the array")
else:
    print("Element at index: ", result)

File: Dart/linearSearch.dart
main() {
  List array = [15, 56, 43, 11, 30];
  int x = 56;
  int result = linearSearch(array, x);

  result == 0
      ? print("Element is not present in array")
      : print("Element is present at index " + result.toString());
}

int linearSearch(List list, int x) {
  for (int i = 0; i < list.length; i++) {
    if (list[i] == x) {
      return i;
    }
  }
  return 0;
}
